packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuanceSessionState.ts

```diff
@@ -4,6 +4,7 @@ export enum OpenId4VcIssuanceSessionState {
   AccessTokenRequested = 'AccessTokenRequested',
   AccessTokenCreated = 'AccessTokenCreated',
   CredentialRequestReceived = 'CredentialRequestReceived',
-  CredentialIssued = 'CredentialIssued',
+  CredentialsPartiallyIssued = 'CredentialsPartiallyIssued',
```

Comment:
We need to somehow handle already stored records. Maybe in the session record we can do an `@Transform` and if the value is `CredentialIssued` we update it to `Completed` for now? Or maybe better to `Error`?

Reply:
not really sure how you to update the record itself